cmd/artifact/main.go

```diff
@@ -0,0 +1,341 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	tcclient "github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/queue"
+	artifact "github.com/taskcluster/taskcluster-lib-artifact-go"
+	"github.com/urfave/cli"
+)
+
+const (
+	ErrInternal = 1
+	ErrBadUsage = 2
+	ErrCorrupt  = 13
+)
+
+// TODO implement an in memory 'file'
+// TODO implement 'redirect' and 'error' artifact types?
```

Comment:
Should these TODOs be in the library package?

Reply:
yep, they probably should be.